css/index.css

```diff
@@ -1,3 +1,305 @@
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
 
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+.container {
+    max-width: 880px;
+    margin: 0 auto;
+}
+
+
+.container .cta {
+	align-content: center;
+	justify-content: space-between;
+}
+
+
+/* Header Styles */
+
+header {
+    display: flex;
+    max-width: 100%;
+  }
+
+header nav {
+    display: flex;
+    width: 691px;
+    vertical-align: top;
+    margin-top: 35px;
+    justify-content: space-between;
+  }
+  
+  header nav a {
+    margin-right: 65px;
+    font-size: 14px;
+    color: gray;
+    text-decoration: none;
+    display: flex;
+  }
+
+
+.header .navbar a:hover {
+    text-decoration: underline;
+    color: black;
+}
+
+.header .navbar .a:first-child {
+    margin-left: 0;
+  }
+  
+.header .navbar a:last-child {
+    margin-right: 0;
+  }
+
+.header .logo {
+    margin-top: 25px;
+ }
+
+
+/* Call To Action (CTA) Styles */
+
+.cta {
+    display: flex;
+    justify-content: flex-end;
+    
+}
+
+.cta .cta-text {
+    font-size: 72px;
+    text-align: center;
+    letter-spacing: 5px;
+    display: flex;
+    justify-content: center;
+    flex-flow: column;
+    align-items: center;
+    margin: 0 82px 0 0;
+}
+
+.cta .cta-text button {
+    border: 1px double black;
+    font-size: 16px;
+    background: white;
+    width: 170px;
+    height: 32px;
+    margin-top: 10px;
+    cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+    background: black;
+    color: red;
+}
+
+.cta img {
+    display: inline-block;
+    margin-top: 69px;
+  }
+
+
+
+/* Main Content Styles */
+
+.main-content {
+   border-top: 2px solid black; 
+   border-bottom: 2px solid black;
+   margin: 75px 0 35px 0;
+   padding: 35px 0;
+   font-size: 16px;
+   line-height: 18px;
+}
+
+.main-content img{
+    display: block;
+    margin: 30px auto;
+}
+
+.main-content .top-content {
+    display: flex;
+
+}
+
+.main-content .middle-img {
+    display: block;
+    margin: 30px auto;
+}
+
+.main-content .bottom-content {
+    display: flex;
+}
+
+.main-content .bottom-content p {
+    padding-right: 20px;
+}
+
+/* Contact Styles */
+
+.contact p {
+    margin: 15px 0;
+}
+
+/* Footer Styles */
+
+footer {
+    text-align: center;
+    margin: 40px 0 20px;
+}
+
+/* Services Styles */
+
```

Comment:
you have css styles for services but I do not see a services.html page. You should be making the HTML page then and only then doing css

Reply:
That copied over form the last great idea project. I only did the index page.